js/graphicView.js

```diff
@@ -0,0 +1,41 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'core/js/views/componentView';
+
+class GraphicView extends ComponentView {
+
+  preRender() {
+    this.listenTo(Adapt, 'device:changed', this.resizeImage);
+
+    this.checkIfResetOnRevisit();
+  }
+
+  postRender() {
+    this.resizeImage(Adapt.device.screenSize, true);
+  }
+
+  checkIfResetOnRevisit() {
+    const isResetOnRevisit = this.model.get('_isResetOnRevisit');
+
+    if (isResetOnRevisit) {
```

Comment:
You can probably invert this into a return early.
```js
if (!isResetOnRevisit) return;
this.model.reset(isResetOnRevisit);
```

It's less nesty and a bit cleaer.